gendox-core-api/src/main/java/dev/ctrlspace/gendox/gendoxcoreapi/converters/ProjectConverter.java

```diff
@@ -0,0 +1,38 @@
+package dev.ctrlspace.gendox.gendoxcoreapi.converters;
+
+import dev.ctrlspace.gendox.gendoxcoreapi.model.Project;
+import dev.ctrlspace.gendox.gendoxcoreapi.model.dtos.ProjectDTO;
+import org.springframework.stereotype.Component;
+
+@Component
+public class ProjectConverter implements GendoxConverter<Project, ProjectDTO> {
+    @Override
+    public ProjectDTO toDTO(Project project) {
+        ProjectDTO projectDTO= new ProjectDTO();
+
+        projectDTO.setId(project.getId());
+        projectDTO.setOrganizationId(project.getOrganizationId());
+        projectDTO.setName(project.getName());
+        projectDTO.setDescription(project.getDescription());
+        projectDTO.setCreatedAt(project.getCreatedAt());
+        projectDTO.setUpdatedAt(project.getUpdatedAt());
+
+
```

Comment:
(see first comment on `class ProjectDTO`)
You can use the ProjectAgentConverter here, like
```java
    ProjectAgentDTO agentDTO = projectAgentConverter.toDTO(project.getProjectAgent());
    projectDTO.setProjectAgentDTO(agentDTO);
```